### PR TITLE
podman: T6598: add custom podman build for version 4.9.5

### PR DESCRIPTION
**Pull request by c-po**
## Change Summary

Add custom build instruction for podman package

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

* https://vyos.dev/T6598

## Component(s) name

podman

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-build/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: packages/podman/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+@NonCPS
+
+// Using a version specifier library, use 'current' branch. The underscore (_)
+// is not a typo! You need this underscore if the line immediately after the
+// @Library annotation is not an import statement!
+@Library('vyos-build@current')_
+
+def pkgList = [
+    ['name': 'libyang',
```
Comment (sever-sever) [inline packages/podman/Jenkinsfile]: Change the package name

Comment (c-po) [reply packages/podman/Jenkinsfile]: Thanks, fixed

**Review by sever-sever — APPROVED** [review]

**Review by jestabro — APPROVED** [review]

**Comment by c-po** [comment]
@MergifyIo backport circinus

**Comment by c-po** [comment]
@MergifyIo backport circinus sagitta

**Pull request merged**